tasks/extract-binaries-from-image/samples/sample_extract-binaries-from-image_TaskRun.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
we had a meeting some days ago and this _samples_ dir can be removed.

Reply:
Deleted!